templates/sfx-amd.js

```diff
@@ -1,6 +1,15 @@
 (function(factory) {
-  if (typeof define == 'function' && define.amd)
-    define(factory);
-  else
+  if (typeof define == 'function' && define.amd){
+    define(function () {
+      var obj = factory.apply(this, arguments);
+      var keys = Object.keys(obj);
```

Comment:
I was planning on just having the first module from the list of modules in the sfx bundle as the exporting module. I think that's a valid tradeoff and keeps things simple. (If I had my way, SFX bundles wouldn't allow addition actually)